README.md

```diff
@@ -13,6 +13,35 @@ bash install-devenv.sh
 **If you are setting up a new development environment, run `devenv bootstrap` after installing.**
```

Comment:
I know this is unrelated to this PR, but I think we should explain that devenv bootstrap should be run in specific repos.

Reply:
i'll change this to "If you are setting up a new laptop, run `devenv bootstrap` after installing."

it's a one-time thing for new engineers, not something that should be run in specific repos